casts/spinnerOverlay/src/SpinnerOverlay/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Platform, StyleSheet, View, Modal } from 'react-native';
+import { WaveIndicator } from 'react-native-indicators';
+
+type Props = {
+    visible: boolean;
+};
+
+const waveConfig = {
+    size: 40,
+    count: 2,
+};
+
+export function UISpinnerOverlay({ visible = false }: Props) {
+    const renderSpinner = React.useMemo(() => {
+        return (
+            <View testID="spinner_loader" style={[styles.container]} key={`spinner~${Date.now()}`}>
+                <View style={styles.background}>
```

Comment:
It looks like we can now use only one View instead of two wrapped one in each other...
What do you think?